zeeguu_api/api/sessions.py

```diff
@@ -108,6 +109,51 @@ def get_anon_session(uuid):
     return str(session.id)
 
 
+@api.route("/send_code/<email>", methods=["POST"])
+@cross_domain
+def send_code(email):
+    """
+    This endpoint generates a unique code that will be used to allow
+    the user to change his/her password. The unique code is send to
+    the specified email address.
+    """
+    code = UniqueCode(email)
+    zeeguu.db.session.add(code)
```

Comment:
i know that most of the API is written like this, 
but I think it's nicer and cleaner if at the top
of the file we simply declare session = zeeguu.db.session
and then we write code like this simply 

    session.add(code)
    session.commit()

Reply:
Totally agree. I used db_session instead to not get confused with the flask session. 

I defined db_session in the init file and updated the other endpoints accordingly.

Reply:
nice idea with the db_session. 
i just realized yesterday that there's so many things called session in the system, that i'm sorry for the dev-team. flask.session where we save the session_id which is the  session column of the Session table... :)